app/src/main/java/org/mozilla/focus/browser/cursor/CursorView.kt

```diff
@@ -38,13 +34,12 @@ class CursorView(context: Context, attrs: AttributeSet) : View(context, attrs) {
     private val radius = context.resources.getDimensionPixelSize(R.dimen.remote_cursor_size) / 2f
     private val paint = Paint().apply {
         style = Paint.Style.FILL
-        alpha = CURSOR_ALPHA
         isAntiAlias = true
-        shader = RadialGradient(radius, radius, /* radius */ radius,
-                ContextCompat.getColor(context, R.color.teal50), ContextCompat.getColor(context, R.color.photonBlue50),
-                Shader.TileMode.CLAMP)
     }
 
+    private val cursorBg = BitmapFactory.decodeResource(resources, R.drawable.cursor_bg)
```

Comment:
nit: can you optimize these assets if you haven't done so already? I use [ImageOptim](https://imageoptim.com/mac)